docs/pages/common_issues.mdx

```diff
@@ -14,4 +14,9 @@ If you are using sentry, API routes returns empty body. You could try configurin
 
 #### My ISR page has this cache-control header `s-maxage=2, stale-while-revalidate=2592000`
 
-Given how ISR works, while waiting for the revalidation to happen, the page will be served using this cache control header. This prevent your server from being overloaded by a lot of requests while the revalidation is done. You can read more about it [here](/inner_workings/isr).
\ No newline at end of file
+Given how ISR works, while waiting for the revalidation to happen, the page will be served using this cache control header. This prevent your server from being overloaded by a lot of requests while the revalidation is done. You can read more about it [here](/inner_workings/isr).
+
+#### Unzipped size must be smaller than 262144000 bytes
+
+AWS Lambda has an unzipped size limit of 250MB. If your app is over this limit, then it is most likely using a node_module library that is too large for serverless or there is a large dev dependency getting bundled.
```

Comment:
Maybe add that what's important is the `.open-next/server-function` folder, and the rest are mostly irrelevant.
We could also add that a big bundle size will increase cold start a lot